src/routes/external.route.ts

```diff
@@ -27,14 +29,25 @@ class ExternalRoute implements Routes {
   private validatorMiddleware = new ValidatorMiddleware();
   private sessionMiddleware = new SessionMiddleware();
   // Controllers
+  private demoRequestController = new DemoRequestController();
   private authenticateController = new AuthenticateController();
   private schoolController = new SchoolController();
 
   constructor() {
+    this.initializeDemoRequestRoutes(`${this.path}/demo-request`);
     this.initializeAuthRoutes(`${this.path}/auth`);
     this.initializeSchoolRoutes(`${this.path}/school`);
   }
 
+  private initializeDemoRequestRoutes(prefix: string) {
+    // API FOR BOOKING DEMO REQUEST
+    this.router.post(
```

Comment:
@hdrayka9060 Create a Additional endpoint which will be called from DAG/Scheduler. will scan all booking for today and will create a meeting ( there is a google api to do this ) . and it will also send remainder email.

if not try to handle creating a meeting link in this api only (/booking). or send common meeting links to all (other can request to join not a preferred way)